draft-ietf-tls-tls13.md

```diff
@@ -2729,6 +2729,10 @@ TLS also allows other messages to be sent after the main handshake.
 These messages use a handshake content type and are encrypted under the application
 traffic key.
 
+Handshake messages sent after the handshake MUST NOT be interleaved with
+application data. That is, if a message is split over two or more handshake
+records, there MUST NOT be any application data records between them.
```

Comment:
This sort of implicitly assumes that there are only application data or handshake records and no other types ... which is basically true for this purpose, but why not just say "MUST NOT be any other records between them"?